habitat-baselines/habitat_baselines/config/rearrange/rl_rearrange.yaml

```diff
@@ -3,6 +3,7 @@
 defaults:
   - /benchmark/rearrange: rearrange
   - /habitat_baselines: habitat_baselines_rl_config_base
+  - /habitat/simulator/sim_sensors@habitat_baselines.eval.extra_sim_sensors.third_rgb_sensor: third_rgb_sensor
```

Comment:
This grabs the sim sensor `/habitat/simulator/sim_sensors: third_rgb_sensor` from the config store and places it in `habitat_baselines.eval.extra_sim_sensors.third_rgb_sensor.third_rgb_sensor`. This is a one line change that will replace the `video_render_views` list. 
Adding new evaluation only sensors is as easy as adding sim sensors to the config.